api/run.sh

```diff
@@ -2,10 +2,9 @@
 
 set -e
 
-while [[ "$(curl --insecure -s -o /dev/null -w '%{http_code}' http://es:9200/)" != "200" ]]
-do
-  echo "Waiting for Elasticsearch connection..."
-  sleep 2
+while [ "$(curl -s -o /dev/null -w '%{http_code}' "http://${ELASTICSEARCH_URL:-es}:${ELASTICSEARCH_PORT:-9200}/_cluster/health")" != "200" ]; do
```

Comment:
Can this line be split up?

```suggestion
es_url = "http://${ELASTICSEARCH_URL:-es}:${ELASTICSEARCH_PORT:-9200}/_cluster/health"
while [ "$(curl -s -o /dev/null -w '%{http_code}' "$es_url")" != "200" ]; do
```

Would this work?

Reply:
It works, except for the spaces around `=` (shell scripts, right?). Fixed in 6df39aa.